documentation/src/main/asciidoc/topics/proc_adding_jgroups_extras.adoc

```diff
@@ -0,0 +1,48 @@
+[id='adding-jgroups-extras_{context}']
+= Adding Additional JGroups Dependencies
+{brandname} includes default JGroups stacks that use discovery protocol implementations that are specific to cloud providers.
+These implementations require artifacts that {brandname} does not include by default.
+
+For this reason, if you plan to use these JGroups stacks, you need to add additional dependencies to your project.
+
+[%header,cols=3*]
+|===
+|File name
+|Discovery protocol
+|Additional dependency
+
+|`default-jgroups-ec2.xml`
+|`NATIVE_S3_PING`
+|`org.jgroups.aws.s3:native-s3-ping`
+
+|`default-jgroups-google.xml`
+|`GOOGLE_PING2`
+|`org.jgroups.google:jgroups-google`
+
+|`default-jgroups-azure.xml`
+|`AZURE_PING`
+|`org.jgroups.azure:jgroups-azure`
+
+|===
+
+.Procedure
```

Comment:
this file is shared between server & embedded. This procedure does not work for the server.